cloudinit/cmd/main.py

```diff
@@ -1033,7 +1037,11 @@ def main(sysv_args=None):
 
     # This subcommand allows you to run a single module
     parser_single = subparsers.add_parser(
-        "single", help="Run a single module."
+        "single",
+        help=(
+            "Manually run a single module. Useful for "
+            "testing during development."
+        ),
```

Comment:
Should we add DEPRECATED to `--force` and `--debug` when called from the CLI too?